hack/verify-manifests.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+DIFFROOT="${REPO_ROOT}/config"
+TMP_ROOT="${REPO_ROOT}/_tmp"
+
+
+cleanup() {
+    rm -rf "${TMP_ROOT}"
+}
+trap "cleanup" EXIT SIGINT
+cleanup
+
+MANIFEST_ROOT=$TMP_ROOT make manifests
+
+echo "diffing ${DIFFROOT} against freshly generated manifests"
+ret=0
+git diff --no-index --diff-filter=MD ${TMP_ROOT} ${DIFFROOT} > /dev/null 2>&1 || ret=$?
```

Comment:
Instead of redirecting to `/dev/null`, could you use the `--stat` flag to at least print a list of the files that it thinks are out of date?

Reply:
sure